src/status_im2/contexts/quo_preview/list_items/saved_address.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im2.contexts.quo-preview.list-items.saved-address
+  (:require [quo2.core :as quo]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [(preview/customization-color-option {:key :account-color})
+   {:key :blur? :type :boolean}
+   (preview/customization-color-option)])
+
+(defn view
+  []
+  (let [state (reagent/atom {:type                :default
+                             :customization-color :blue
+                             :account-color       :pink
+                             :title               "Alisher Yakupov"
```

Comment:
The descriptor should allow the dev/designer to at least change `title` as well.

Reply:
Added 👍